frontend/www/js/omegaup/components/certificate/Details.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h4 class="panel-title">
+        {{ T.certificateDetailsTitle }}
+      </h4>
+    </div>
+    <div class="panel-body">
+      {{ ui.formatString(T.certificateDetailsBody, { uuid: this.uuid }) }}
```

Comment:
```suggestion
      {{ ui.formatString(T.certificateDetailsBody, { uuid: uuid }) }}
```

en el Vue, el `this` va implícito.

Reply:
Parece que ya sólo falta esto!

Reply:
```suggestion
      {{ ui.formatString(T.certificateDetailsBody, { uuid: urlencode(uuid) }) }}
```

sabes qué? se me olvidó el [`urlencode()`](https://www.php.net/manual/en/function.urlencode)!

Reply:
Gracias, acabo de agregar este cambio 👍